### PR TITLE
Brett/dev/accept ptp

### PR DESCRIPTION
**Pull request by berendo**

### CODE REVIEW

File: FreeRTOS_IP.c [hunk 1]
```diff
@@ -1377,6 +1386,20 @@ eFrameProcessingResult_t eReturned = eReleaseBuffer;
...
+			case ipPTP_FRAME_TYPE:
+				/* The Ethernet frame contains an PTP packet. */
+				if( pxNetworkBuffer->xDataLength >=
+                    (ipEXPECTED_EthernetHeader_t_SIZE + ipEXPECTED_PTPHeader__SIZE))
+				{
+					eReturned = ePTPProcessPacket( pxNetworkBuffer->pucEthernetBuffer );
+				}
+				else
+				{
+                    printf("PTP Packet too small, %d\n", pxNetworkBuffer->xDataLength);
```
Comment (sabhiram) [inline FreeRTOS_IP.c:1398]: I think your tabs are goofed. Could you make sure you run lint on this? `make format`

**Review by sabhiram — COMMENTED** [review]
I just realized that this repo does not have the likes of `make format`. Could you please fix your tabs and space issue and push again? The whitespace issue should be obvious from the code diff.

**Review by eechengc — APPROVED** [review]
It looks good.

File: FreeRTOS_IP.c (hunk 1 above)
Comment (brudley) [reply FreeRTOS_IP.c:1398]: Yup. This repo of external code is not in our code style so using formatter results in  massive changes which we don't want to do.    Will follow up to adjust to this repos's format (eg. uses  actual tabstops vs spaces).

**Pull request merged**